src/App.js

```diff
@@ -1,26 +1,25 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { Component } from "react"
+import "./App.css"
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import {
+  BrowserRouter as Router,
+  Route,
+} from "react-router-dom";
+
+import Home from './pages/Home';
+
+class App extends Component {
+  render() {
+    return (
+      < Router >
+        <Route exact path="/">
```

Comment:
Hmm, I don't think the path needs to be exact. I think by default even if you go to https://dreamerscholars.web.app/sdf it should redirect to the home page

Reply:
Alright meaning just 
<Route>
            <Home />

Reply:
<Route> without the exact path part ?

Reply:
Yes I believe so: